services/frontend/src/components/layouts/appLayout/topNav/TopNavView.jsx

```diff
@@ -47,7 +47,11 @@ const TopNavView = ({ isAdmin, loading, displaySearch, displayLogo, intl }) => {
 
   // menu options for profile dropdown
   const menu = (isDropdown, optionalStartMenuItems) => (
-    <Menu className={isDropdown ? "dropDownMenu" : "hamburgerMenu"}>
+    <Menu
+      tabIndex={-1}
+      tabindex="-1"
```

Comment:
I would maybe put a small comment here explaining why this is done, and you'll need to fix the linting errors (https://cicd.ised-isde.canada.ca/blue/organizations/jenkins/DSD%2FI-Talent%2FI-Talent/detail/user-dropdown-accessibility/9/pipeline), for this one here, you'll just need to disable eslint for these two lines for the `react/jsx-no-duplicate-props` rule. Maybe something like `/* eslint-disable react/jsx-no-duplicate-props */` then this afterwards `/* eslint-enable react/jsx-no-duplicate-props */`

Reply:
Tab index at -1 is used for off-screen content that appears on a specific event. The element is not reachable via sequential keyboard navigation, however, we focused the dropdown with Javascript instead so it works for tabbing and normal clicking.